---
layout: post
title: The Curious Case of the Tasmanian Tiger
comments: true
---

A few months ago, I became interested in the study of ancient DNA. Think this dabbling primarily arose after I stumbled upon a [TED](https://www.ted.com/talks/svante_paeaebo_dna_clues_to_our_inner_neanderthal) talk given by *Dr. Svante Paabo*. This in turn led me to reading a book written by him - which I found to be an interesting concoction of best practices in research and a detailing of the kind of man he is.

Here is an excerpt from the book:
> "To me, DNA sequences seemed to offer a much better chance to correctly gauge how species were related to each other. DNA sequences can accumulate hundreds and thousands of mutations over time, each of which occurs independently of one another, and most of which have no influence on how an orgainsm looks or behaves. In contrast, measurement of morphological features necessarily is done on traits that might very well affect the survival of the organism, and the sizes of different features, such as various bones, might be linked to one another. Because of the greater number of independent, randomly varying data points that can be accumulated, DNA sequences allow for reconstruction of relationships with greater rigor than morphological features. In fact, in contrast to morphological features, even the timing of divergences from a common ancestor can be derived from the number of differences that have accumulated in DNA sequences, since these differences occur roughly as a function of time, at least within a group of related animals."

Paabo talks about some of his initial projects before he started working with the Neanderthal genome. One of these was sequencing the Mitochondrial DNA of the Tasmanian Tiger, a carnivorous marsupial which became extinct in the 20th Century. 


<p><figure><img src="/images/2017-12-10/thylacine.jpg" alt=""/><figcaption>
   [Thylacinus or Tasmanian Tiger, in Washington D.C. National Zoo, c. 1906. Source: Photograph by E.J. Keller, from the Smithsonian Institution archives (via Wikipedia)]</figcaption></figure></p>

Called a *tiger*, probably because of the stripes on its tail, I think it was said to be morphologically closer to a dog (than a tiger). In fact, some anecdotes seemed to suggest that it was hard to differentiate between the skeleton of a dog and a Tasmanian Tiger, even for students of medicine. These ponderings took me to the idea of Convergent Evolution, wherein species from two distant families develop similar morphological traits (for example, the absence of placental birth in Tasmanian Tigers and kangaroos!) because of the environmental factors.  

I wonder if one could specifically pinpoint the beginning of alteration of DNA arising from an environmental event (such as climate change)? Possibly there would be models for including environmental factors which are used in the field of population genetics - but think it would be cool to know how accurate they are and to use such models for predicting the changes in the DNA of current-day humans in response to hypothetical, climate-change scenarios. 

I also learnt that while studying ancient DNA, such as that of a Tasmanian Tiger specimen (Tasmanian Tigers became extinct fairly recently. I guess considering that ancient DNA involves beings who were alive 30,000 years ago or more, Tasmanian Tigers are the 'new' inductees in this field!), it is preferable to only consider short nucleotide sequences (a few hundred base pairs), since that eliminates error while ampilfying DNA. I wonder what are the general metrics to compare two such short sequences - for example, are metrics like *R2* used to suggest how well one sequence fits the other?

Guess, a [course](https://www.coursera.org/learn/dna-sequencing) in Algorithms for DNA Sequencing would help in understanding how the DNA Sequence data is pre-processed and analysed!   